user-console/src/main/java/org/pentaho/mantle/client/admin/UserRolesAdminPanel.java

```diff
@@ -186,6 +234,7 @@ private Widget createUsersPanel() {
     availableRolesPanel.add( availableRolesListBoxWrapper );
     availableRolesListBox.setVisibleItemCount( 20 );
     availableRolesListBox.addStyleName( "users-roles-selection-list" );
+    availableRolesListBox.getElement().setId( "available-roles-list" );
```

Comment:
Isn't the `users-roles-selection-list` CSS class enough?
Also, it is not considered good practice to target CSS rules at element ids (which also applies to all of the identifiers introduced by this PR).

Reply:
what's wrong in having id ?

Reply:
I can change the css to use class instead though

Reply:
Element ids have a very high specificity when used as CSS selectors, making it harder, and in some cases, impossible, to be overridden using non-id CSS selectors.

Reply:
it's already changed to use the class selector. but it's better to have just ids for elements anyway.

Reply:
> but it's better to have just ids for elements anyway.
What do you mean by better? Classes and ids each have their own uses.

Reply:
as mentioned earlier in css file it's changed from id selector to class selector.
IDs as you mentioned has it's purpose and uses. by better I probably meant the same.